tests/fixtures/analog/commands.js

```diff
@@ -0,0 +1,420 @@
+export const uplinkCommands = {
+    'new event': {
+        hex: {
+            dump: '1506081e000973cbe1',
```

Comment:
```suggestion
            dump: '15 06 08 1e 00 09 73 cb e1',
```

?

and in other places

Reply:
in this form, we get a dump from the device, and since the search is performed using getByText and since there is no test-id, the values ​​of the entered dump will be the same in the logs, it is more difficult to search for the desired element